bazel/repository_locations.bzl

```diff
@@ -322,26 +322,26 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_desc = "Data Collect Protocols of Apache SkyWalking",
         project_url = "https://github.com/apache/skywalking-data-collect-protocol",
         name = "skywalking_data_collect_protocol",
-        sha256 = "fa9ac679624217f30b6e8d5c450365386c610e2d08188a20f0340c3b14401833",
-        urls = ["https://github.com/apache/skywalking-data-collect-protocol/archive/v8.3.0.zip"],
-        strip_prefix = "skywalking-data-collect-protocol-8.3.0",
-        version = "8.3.0",
+        sha256 = "4a3401098e0e49ae56d03cc4890b21c75bab6ccf1c157d73bb0c93fc15bb07c7",
+        urls = ["https://github.com/apache/skywalking-data-collect-protocol/archive/{version}.tar.gz"],
```

Comment:
Use latest release here? https://github.com/apache/skywalking-data-collect-protocol/releases/tag/v8.4.0

Reply:
The latest version of cpp2sky implements feature that doesn't be implemented in v8.4.0.